spec/requests/api/v2/providers/courses/publish_spec.rb

```diff
@@ -173,7 +173,7 @@
               "/data/attributes/how_school_placements_work",
               "/data/attributes/course_length",
               "/data/attributes/fee_uk_eu",
-              "/data/attributes/required_qualifications",
+              nil,
```

Comment:
This isn't correct but it's how the serialiser is currently behaving. I'll card up to fix in another PR.